src/pip/_internal/utils/setuptools_build.py

```diff
@@ -103,7 +103,8 @@ def make_setuptools_develop_args(
     if prefix:
         args += ["--prefix", prefix]
     if home is not None:
-        args += ["--home", home]
+        # args += ["--home", home]
```

Comment:
Why didn't you just drop the comment?

Reply:
Actually no real reason - a kind of 'this is a wip-pull request' kind of thing? Will drop.

Reply:
Oh, so maybe mark it as a draft then?